packages/openneuro-app/src/scripts/faq/faq.jsx

```diff
@@ -22,10 +24,15 @@ class Faq extends React.Component {
     })
 
     return (
-      <div className="container faqs">
-        <h1>{"FAQ's"}</h1>
-        {faqs}
-      </div>
+      <>
+        <Helmet>
+          <title>FAQ - {pageTitle}</title>
```

Comment:
I can't think of something better but expanding on FAQ somehow might be good to make sure it has search visibility.